Clean up `Client/Interface/Multiplayer` (#9513)
ppy/osu-wiki: wiki/Client/Interface/Multiplayer/en.md

# Multiplayer

> **Multiplayer** (sometimes shortened to *Multi*) is a mode in which up to 16 players can compete against each other or as a team on maps decided by the host. Players can also complete maps together depending on host's settings.

Reviewer: would rather have this combined into one sentence if it absolutely must be mentioned

```suggestion
**Multiplayer** (sometimes shortened to *Multi*) is a mode in which up to 16 players can compete against each other individually or in teams, or play together co-operatively, on maps decided by the host.
```

Covering the middle of the screen is a list of available matches.

> Most matches have light white backgrounds, indicating that they were created normally in [this way](#creating-a-new-game). While there are also ones with purple backgrounds, which were **tournament matches** created by using `!mp make` or `!mp makeprivate`.

Reviewer: avoid links that say "this link" or "here", because at a glance it's less obvious what it's linking to. the writing should be the same whether there is a link or not.

```suggestion
Most matches have light white backgrounds, indicating that they were [created normally through the game UI](#creating-a-new-game). There are also ones with purple backgrounds, which are *tournament matches* created and managed using [lobby management chat commands](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands) such as `!mp make` or `!mp makeprivate`.
```

![](img/multi-match-header.jpg "Example of the header")

> The top-left text indicates whether the player is the match host or a normal player. In the middle, there is a player box with information about [performance points](/wiki/Performance_points/Total_performance_points), [overall accuracy](/wiki/Gameplay/Accuracy), [level](/wiki/Gameplay/Score/Total_score#level), [rank](/wiki/Ranking#performance-points-ranking) and the selected [game mode](/wiki/Game_mode). To the right is a button for pausing the currently playing song. Pressing the button again can resume the song.

Reviewer: unnecessary

```suggestion
The top-left text indicates whether the player is the match host or a normal player. In the middle, there is a player box with information about [performance points](/wiki/Performance_points/Total_performance_points), [overall accuracy](/wiki/Gameplay/Accuracy), [level](/wiki/Gameplay/Score/Total_score#level), [rank](/wiki/Ranking#performance-points-ranking) and the selected [game mode](/wiki/Game_mode). To the right is a button for pausing the currently playing song.
```

### Match setup buttons

There are two large wide orange and blue buttons above the [chat console](/wiki/Client/Interface/Chat_console).

> The orange `Leave Match` button on the left is self-explanatory. If the match host leave the match, the host will be automatically transferred to the remaining player in slot order. If there is no one in the match, it will automatically close, except for ones made by `!mp` commands, which will instead close after 30 minutes.

Reviewer: ```suggestion
The orange `Leave Match` button on the left is self-explanatory. If the match host leave the match, the host will be automatically transferred to the remaining player in slot order. If there is no one in the match, it will automatically close, except for ones made by [`!mp` commands](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands), which will instead close after 30 minutes.
```

#### Health

When a player's health bar has been depleted completely, they can still continue playing, but the score will be considered as failed and will not appear on leaderboards. The player can be revived if they reach full health again, unless they use [Sudden Death (SD)](/wiki/Gameplay/Game_modifier/Sudden_Death).

> In Team VS matches, being in the failed state at the end of the match will make the score not contribute to the team score. If all players of a team have failed, the current gameplay immediately ends, and the opposite team wins by default.

Reviewer: "gameplay" is not a term that refers to any individual game or match

```suggestion
In Team VS matches, being in the failed state at the end of the match will make the score not contribute to the team score. If all players of a team have failed, the match immediately ends and the opposite team wins by default.
```

All players share the same health bar. If it depletes completely, the match ends. In Tag-team VS, each team has their own health bars, and losing all health will cause that team to lose.

> Players using the [Sudden Death (SD)](/wiki/Gameplay/Game_modifier/Sudden_Death) game modifier will cause the whole team to fail if they fail due to it. In Tag-team VS mode, this cause both teams's current game to end immediately.<!-- TODO: What's about EZ? (from former comments) -->

Reviewer: if that's the case then the explanation can be simplified a bit

```suggestion
Players using the [Sudden Death (SD)](/wiki/Gameplay/Game_modifier/Sudden_Death) game modifier will cause the whole match to end if they fail due to it, even in Tag-team VS.<!-- TODO: What's about EZ? (from former comments) -->
```

> Players using the [Sudden Death (SD)](/wiki/Gameplay/Game_modifier/Sudden_Death) game modifier will cause the whole match to end if they fail due to it, even in Tag-team VS. If [Easy (EZ)](/wiki/Gameplay/Game_modifier/Easy) game modifier is used, players can still get two extra lives. But in Tag-team VS, when a team's health drops to zero, though the [health bar](/wiki/Client/Interface/Health_bar) will be refilled, the match will judge them as **failed** and end immediately.

Reviewer: ```suggestion
Players using the [Sudden Death (SD)](/wiki/Gameplay/Game_modifier/Sudden_Death) game modifier will cause the whole match to end if they fail due to it, even in Tag-team VS. If [Easy (EZ)](/wiki/Gameplay/Game_modifier/Easy) is used, players can still get two extra lives. However, in Tag-team VS, when a team's health drops to zero, even though the [health bar](/wiki/Client/Interface/Health_bar) will be refilled, the match will judge them as *failed* and end immediately.
```

For `Tag co-op`, the result shown is the total score, the [judgements](/wiki/Gameplay/Judgement) and accuracy of the team. Individual results are not shown, and can only be seen in the match history. The result screen only appears if the beatmap was cleared successfully, unlike other multiplayer modes.

> The `Tag-team VS` results screen is identical to the [`Team VS`'s results screen](#team-vs-results-screen), but individual results can be seen under different tabs of each team.

Reviewer: ```suggestion
The `Tag-team VS` results screen is identical to the [`Team VS`'s results screen](#team-vs-results-screen), except that individual results can be seen under different tabs for each team.
```
